apps/openmw/mwrender/actoranimation.cpp

```diff
@@ -62,6 +62,7 @@ ActorAnimation::~ActorAnimation()
     }
 
     mScabbard.reset();
+    mHolsteredShield.reset();
```

Comment:
Just curious: Why is there no mHolsteredWeapon reset?

Reply:
Because there is no `mHolsteredWeapon` in OpenMW. Holstered weapons is handled in the `mScabbard`.